game-core/src/main/java/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -648,19 +672,35 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
         .build();
   }
 
-  public static Predicate<Unit> defendingBombingRaidInterceptors(final PlayerID attacker, final GameData data) {
-    return PredicateBuilder.of(
+  /**
+   * Returns a unit predicate that determines if it can potentially intercept including checking any
+   * air base requirements.
+   */
+  public static Predicate<Unit> defendingBombingRaidInterceptors(final Territory territory, final PlayerID attacker,
+      final GameData data) {
+    final Predicate<Unit> canIntercept = PredicateBuilder.of(
         Matches.unitCanIntercept())
         .and(Matches.unitIsEnemyOf(data, attacker))
         .and(Matches.unitWasInAirBattle().negate())
         .andIf(!Properties.getCanScrambleIntoAirBattles(data), Matches.unitWasScrambled().negate())
         .build();
+    final Predicate<Unit> airbasesCanIntercept = Matches.unitIsEnemyOf(data, attacker)
+        .and(Matches.unitIsAirBase())
+        .and(Matches.unitIsNotDisabled())
+        .and(Matches.unitIsBeingTransported().negate());
+    return u -> {
+      boolean result = canIntercept.test(u);
```

Comment:
Bit of a style nit, I suppose the following:
```
      boolean result = canIntercept.test(u);
      if (Matches.unitRequiresAirBaseToIntercept().test(u)) {
        result = result && Matches.territoryHasUnitsThatMatch(airbasesCanIntercept).test(territory);
      }
      return result;
```
Can be condensed:

```
      return u -> canIntercept.test(u);
          && Matches.unitRequiresAirBaseToIntercept().test(u))
          && Matches.territoryHasUnitsThatMatch(airbasesCanIntercept).test(territory);
```

Reply:
Updated with correct expression.